### PR TITLE
load model eic

### PR DESCRIPTION
**Pull request by Reusdd**
## Checklist

- [ ] Format your code according to the [Format code with pre-commit](https://docs.sglang.ai/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [ ] Add unit tests according to the [Run and add unit tests](https://docs.sglang.ai/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [ ] Update documentation according to [Write documentations](https://docs.sglang.ai/developer_guide/contribution_guide.html#write-documentations).
- [ ] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.ai/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.ai/developer_guide/contribution_guide.html#benchmark-the-speed).

### CODE REVIEW

File: python/sglang/srt/model_loader/loader.py [hunk 1]
```diff
@@ -1491,6 +1658,297 @@ def _load_model_from_remote_fs(
...
+        model: nn.Module,
+        client: EICConnector,
+        model_config: ModelConfig,
+    ) -> nn.Module:
+        start_time = time.perf_counter()
+        tp = get_tensor_model_parallel_world_size()
+        rank = get_tensor_model_parallel_rank()
+        pp = get_pp_group().world_size
+        pp_rank = get_pp_group().rank_in_group
+        model_name = client.url.removeprefix("eic://")
```
[the remark this answers is not in the document]
Comment (Reusdd) [reply python/sglang/srt/model_loader/loader.py:1706]: 了解，这个我加上

File: python/sglang/srt/model_loader/loader.py [hunk 2]
```diff
@@ -1382,11 +1391,22 @@ def load_model(
 
 class RemoteModelLoader(BaseModelLoader):
     """Model loader that can load Tensors from remote database."""
-
+    BATCH_SET_SIZE = 20 * 1024 * 1024
```
[the remark this answers is not in the document]
Comment (Reusdd) [reply python/sglang/srt/model_loader/loader.py:1394]: 这个也考虑过，eic.py里主要保留了基础的读写，和模型加载有关的还是留在了loader.py中，这个我再看一下

**Review by rzwei — APPROVED** [review]
